source/jormungandr/jormungandr/scenarios/helper_classes/complete_pt_journey.py

```diff
@@ -49,8 +48,10 @@ def wait_and_get_pt_journeys(future_pt_journey, has_valid_direct_paths):
     if pt_journeys and pt_journeys.HasField(b"error"):
         if pt_journeys.error.id == response_pb2.Error.error_id.Value('no_solution') and has_valid_direct_paths:
             pt_journeys.ClearField(b"error")
+        elif pt_journeys.error.id == response_pb2.Error.error_id.Value('date_out_of_bounds'):
+            raise InvalidDateBoundException(pt_journeys)
         else:
-            raise PtException(pt_journeys)
+            return None
```

Comment:
Does that mean that, now, errors that aren't 'no_solution' or 'date_out_of_bounds' are ignored ?

Reply:
Yes! In this context, we want distributed to continue and return whatever it has found